docs/compilation/composing_functions_with_modules.md

```diff
@@ -43,7 +49,7 @@ x_dec = CounterFhe.inc.decrypt(x_enc)
 assert x_dec == 15
 ```
 
-## Multi inputs, multi outputs
+## Multi inputs/ outputs
```

Comment:
shouldn't we have a space here? ie, either "Multi inputs / outputs" or "Multi inputs/outputs" (I prefer the first)